src/sentry/api/serializers/models/userreport.py

```diff
@@ -48,11 +37,6 @@ def serialize(self, obj, attrs, user):
             'comments': obj.comments,
             'dateCreated': obj.date_added,
             'user': attrs['event_user'],
-            'event': {
-                'id': six.text_type(attrs['event']['id']) if attrs['event']['id'] else None,
-                'eventID': obj.event_id
-            }
```

Comment:
Does this change the API response in a way that could impact clients?

Reply:
Yeah, I can update to keep this in here

Reply:
I think it will impact this endpoint 
https://docs.sentry.io/api/projects/get-project-user-reports/